padle/view/globalstats_view.py

```diff
@@ -0,0 +1,92 @@
+from discordmenu.embed.components import EmbedMain, EmbedThumbnail
+from discordmenu.embed.view import EmbedView
+from tsutils.menu.components.config import UserConfig
+from tsutils.menu.components.footers import embed_footer_with_state
+from tsutils.menu.view.view_state_base import ViewStateBase
+from tsutils.tsubaki.links import MonsterImage
+from tsutils.tsubaki.monster_header import MonsterHeader
+
+
+class GlobalStatsViewState(ViewStateBase):
+    VIEW_STATE_TYPE: str = "GlobalStatsView"
```

Comment:
can you add Padle to the VIEW_STATE_TYPE since those are recognized across all of tsubaki (no need to rename the class, just the class attribute)